raiden/api/rest.py

```diff
@@ -1650,6 +1652,32 @@ def settlement_light(self,
             )
             return result
 
+        message = LightClientMessageHandler.get_light_client_protocol_message_by_internal_identifier(
+            internal_msg_identifier=internal_msg_identifier,
+            wal=self.raiden_api.raiden.wal
+        )
+
+        if not message:
+            return ApiErrorBuilder.build_and_log_error(
+                errors="Light Client Message with internal_msg_identifier = {} not found"
+                       .format(internal_msg_identifier),
+                status_code=HTTPStatus.BAD_REQUEST,
+                log=log
+            )
+
+        if message.is_signed:
+            return ApiErrorBuilder.build_and_log_error(
+                errors="Failed trying to settle a channel that's already settled",
```

Comment:
I think this err should be that the message is already signed,not that the channel is already settled. The settlement may not happen cause an error.

Reply:
yes that's true i did it like this to avoid having to add a new error handling on the SDK but we can speak with @debrayperalta to add that to the SDK and change this message

Reply:
i agree with @marcosmartinez7